widgetastic_manageiq.py

```diff
@@ -878,3 +878,144 @@ def last_page(self):
     @property
     def items_amount(self):
         return self.paginator.page_info()[1]
+
+
+class Stepper(Widget):
+    """ A CFME Stepper Control
+
+        stepper = Stepper(locator='//div[contains(@class, "timeline-stepper")]')
+        stepper.inrease()
+    """
+    def __init__(self, parent, locator, logger=None):
+        super(Stepper, self).__init__(parent=parent, logger=logger)
+
+        self._stepper_control = self.browser.element(locator, parent=parent)
+
+        self.minus_button = Button(self, '-')
+        self.plus_button = Button(self, '+')
+        self.value_field = Input(self, locator='.//input[contains(@class, '
+                                               '"bootstrap-touchspin")]')
+
+    def __locator__(self):
+        return self._stepper_control
+
+    def read(self):
+        return int(self.value_field.read())
+
+    def decrease(self):
+        self.minus_button.click()
+
+    def increase(self):
+        self.plus_button.click()
+
+    def set_value(self, value):
+        if value < 1:
+            raise ValueError('The value cannot be less than 1')
+        steps = int(value) - self.read()
```

Comment:
Mike, I added value < 1 check only because stepper cannot be set to less than 1 in our case. if script tries to set the value less, this means that script has an error.

I agree with you that input has to be converted to int.
But I don't want to add more input checks because this will be either redundant code or mask real errors. We can discuss subject if you wish.